spec/integration/publish_intent_crud_spec.rb

```diff
@@ -23,14 +23,11 @@
         expect(intent.publish_time).to match_datetime(publish_time)
       end
 
-      it "responds with a created status, and the intent as json" do
+      it "responds with a created status and an empty JSON document" do
         put_json "/publish-intent/vat-rates", data
 
         expect(response.status).to eq(201)
-
-        data = JSON.parse(response.body)
-        expect(data['base_path']).to eq('/vat-rates')
-        expect(data['publish_time']).to match_datetime(publish_time)
+        expect(response.body).to eq('{}')
```

Comment:
I'm thinking whether there's any value in returning an empty JSON document in the body?

For consumers of this API, it would be better to receive a `HEAD 201` with no body, instead of seeing a body that has no meaning. This also brings it closer to commonly followed API convention of returning a `HEAD 201` with a location header pointing to the newly created resource (which we don't need) in response to a PUT.